### ISSUE TITLE
Unstable channel hasn't updated for 2 weeks

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#70732]
https://github.com/NixOS/nixpkgs-channels is stuck at 2436c27541b2f52deea3a4c1691216a02152e729. According to https://hydra.nixos.org/job/nixos/trunk-combined/tested#tabs-constituents, the main problem appears to be `nixos.ova`. Does anyone know why build fails?

**Comment by madjar** [comment]
The logs mention `Not enough space left in . (3146537472 KB) to decompress ./VBoxLinuxAdditions.run (8404 KB)`. Out of space?

Is there some publicly available monitoring to confirm this?

**Comment by hedning** [comment]
ova is fixed now: https://github.com/NixOS/nixpkgs/issues/70442, https://github.com/NixOS/nixpkgs/pull/70027

virtualbox guest additions is also fixed: https://github.com/NixOS/nixpkgs/issues/70470

**Comment by worldofpeace** [comment]
Yeah, I'm pretty sure @vcunat had this resolved.

**Comment by tobim** [comment]
Looks good now: https://hydra.nixos.org/build/102761912

**Comment by rpearce** [comment]
This seems to still be an issue, right?
<img width="857" alt="image" src="https://user-images.githubusercontent.com/592876/66588483-1908f400-eb7c-11e9-98df-b3b03ed25e7c.png">
(from https://howoldis.herokuapp.com/)

And all `X`s here going back to 2019-09-29:
https://hydra.nixos.org/job/nixpkgs/trunk/unstable/all

**Comment by worldofpeace** [comment]
It seems to be a new issue with the ipv6 test https://hydra.nixos.org/build/102794246 failing non-deterministically.

**Comment by hedning** [comment]
nixpkgs-unstable is blocked by busybox not building in stdenvBootstrapTools: https://hydra.nixos.org/build/102298542

Looks like 5ea4c9184bc041707483803595a9c767181b6018 reintroduced something which was previously reverted in 64d821d9f4256fbf1215beaed27f30959d4cca39. I'm able to build stedenvBootstrapTools without 5ea4c9184bc041707483803595a9c767181b6018 at least.

**Comment by vcunat** [comment]
I commented directly there: https://github.com/NixOS/nixpkgs/commit/5ea4c9184b#commitcomment-35470680

... but it's a different channel really, as Peter clearly meant nixos-unstable which is considered of much higher importance.

**Comment by rpearce** [comment]
Looks like we're back in business:

<img width="880" alt="image" src="https://user-images.githubusercontent.com/592876/66708902-0fcc8280-ed48-11e9-886d-5cfe67e226e7.png">
